### ISSUE TITLE
Add Disclaimer regarding Agreement regarding collection and use of personal information

### ISSUE THREAD

**Issue by AUSROGER** [anjosh/htm-project#66]
Here is text that should appear after a person first registers to use the app

1. *Purpose of collection and use:* The personal information you are providing will be used by the Family Federation for World Peace and Unification (USA) (hereafter FFWPU USA) for managing records of FFWPU Leaders and lists of Heavenly Tribal Messiahs (HTMs), approving accounts and managing the confirmation of HTMs, managing lists of Tribe Members and their accounts, and for statistical analysis.

2. *Personal data to be collected:* Name, date of birth, nationality, email address and photos of FFWPU leaders and heavenly tribal messiahs. Name, date of birth, nationality, occupation, address, telephone number, email address and photos of heavenly tribal messiah tribe members.

3. *Period of data storage and use:* From the date that data is collected until the day the membership account on the website is terminated.

4. The personal information of Heavenly Tribal Messiah tribe members may be entered on the site only after the Heavenly Tribal Messiah who is managing the information on behalf of the tribe members receives a signed Blessing Registration form from the tribe members who have received the Blessing acknowledging that they are opting in to receive communications.

5. Applicants have the option to decline to provide their personal information. However, if they decline, they will not be able to sign up for a membership account on the website.

Agree  |  Disagree
I am not sure if we need to also have a form here regarding sharing information with a 3rd party? Is this information going to be shared with Korea or other organizations?

**Comment by anjosh** [comment]
Thank you Roger. We need a design for this now. @mikiekwoods could you think about this? Let's discuss in our next call.

**Comment by anjosh** [comment]
@AUSROGER @mikiekwoods where do you think this should appear? I'm thinking somewhere in the app (not part of the login app). Maybe this can be part of the signup process where you also specify your spouse, location, local church community.

**Comment by anjosh** [comment]
I think we should distinguish here that we're not planning on sharing any of the tribe members' (aka Contact aka Carl) personal information. We're only going to use their information for statistics. The Tribal Messiah's (Tim) personal information will be used. So I think for our app only the personal information of someone that registers in the app will be used.

**Comment by noahross72** [comment]
We should be careful about this.  We are also keeping CRM information like blessing dates and other activities.  I suggest keeping it simple and not tie our own hands in the future.  Perhaps we need to talk with Legal.  The main thing is we won't sell or distribute PI to other organizations.

**Comment by anjosh** [comment]
@AUSROGER @noahross72 @mikiekwoods per our previous discussion. I will send an email to HQ and cc us to further discuss this.

**Comment by anjosh** [comment]
adding @mikiekwoods post in slack here:

> Michael Holmes [2:41 PM]
Maybe for HTMs in the field, we can give them an official form they can fill out and the contact actually checks off a box that says something like "I understand that my data will only be shared within the organization (HSA-UWC USA & HSA-UWC) for reporting & statistical research."

> Maybe this too???
"I understand that I am NOT opt-ed in to any subscription list but the representative I met may communicate with me as a personal contact." (edited) 

> Also, if people do opt-out, perhaps we can still have a way to report the number at least?

**Comment by anjosh** [comment]
Olga, from HQ Legal, has created a privacy policy. I've emailed her to review this discussion to make sure that covers everything we need for this. Here's the [Privacy Policy](https://app.zenhub.com/files/112847421/51404d34-4156-4661-b229-1533d883e67d/download).

**Comment by noahross72** [comment]
Josh / Olga

 

This seems to say that if we want to enter our Spiritual child in the app for our own use we need to get permission.  Is that true?  When I enter a contact to Outlook or Gmail I don’t do this.

 

Noah

**Comment by anjosh** [comment]
Olga explained to me that this is there to protect us. Saying that we, creators of MyTribe, are not liable for users putting in information that they don't have permission to put in. This is likely what Google and Outlook have in their privacy policy as well.

**Comment by noahross72** [comment]
Mr. Ross,

The policy requires the user to obtain the guest's permission. This is the
only way to ensure that HSa fulfills its privacy obligations.
Permission does not have to be in any particular form. Even oral okay would
suffice.

*Olga Majitova*
HSA-UWC
212-719-4980
212-719-4996
4W 43rd St
New York NY 10036

**Comment by noahross72** [comment]
Hi Olga,

 

In my company, we use a CRM web tool to keep track of sales contacts.  We don’t ask permission because it is just a tool instead of keeping it on a spreadsheet.

 

I’m afraid that asking TMs to ask permission to do this will kill the whole usefulness of the app.

 

Noah

**Comment by noahross72** [comment]
Also, when you put contact information in your Google contacts or Outlook, you don’t tell everybody.

 

From: Noah Ross <noahross72@gmail.com> 
Sent: Wednesday, August 21, 2019 5:38 PM
To: 'Olga Majitova' <omajitova@unification.org>
Cc: 'anjosh/htm-project' <reply@reply.github.com>
Subject: RE: [anjosh/htm-project] Add Disclaimer regarding Agreement regarding collection and use of personal information (#66)

 

Hi Olga,

 

In my company, we use a CRM web tool to keep track of sales contacts.  We don’t ask permission because it is just a tool instead of keeping it on a spreadsheet.

 

I’m afraid that asking TMs to ask permission to do this will kill the whole usefulness of the app.

 

Noah

**Comment by noahross72** [comment]
Mr Ross,

The policy requires the user to obtain permission to collect the
information from their guests, not specifically to store it anyway.
GDPR guidelines which we are trying to comply with are very clear, and this
small provision in the policy barely helps us to fulfill our obligations.
According to my (very limited) understanding of the CRM web tools, they do
not exclude any privacy obligations of HSA. There is a bit about that here:
https://www.superoffice.com/blog/gdpr-crm/

*Olga Majitova, Esq.*
HSA-UWC
212-719-4980
212-719-4996
4W 43rd St
New York NY 10036

**Comment by anjosh** [comment]
Olga,

Could we re-word it to something like what the Infusionsoft CRM's privacy
policy says: https://keap.com/legal/application-privacy-notice

This Notice does not apply to personal data we collect by other means, such
> as personal data that we receive directly through Keap’s own publicly
> accessible websites. In addition, our customers use our hosted technology
> platform to store and process data of their own users and customers at
> their own discretion. In this circumstance, we act only as a storage and
> service provider, without any interest in or knowledge about what is being
> stored. In general, we only access such personal data at our customer’s
> request in connection with customer support or account administration
> matters, as is reasonably necessary in order to provide the services that
> our customer has directed us to provide, or as may be required by law.


Thank you for your help with this.

Josh

**Comment by anjosh** [comment]
Josh,

I cannot do that unless you confirm that this statement is actually true.
Can you confirm that "we act only as a storage and service provider,
without any interest in or knowledge about what is being stored"?  Won't
information about the users' guests be used at least for statistical
purposes?

*Olga Majitova, Esq.*
HSA-UWC
212-719-4980
212-719-4996
4W 43rd St
New York NY 10036

**Comment by anjosh** [comment]
We might be able to live with that, although TMs today send the same information to HQ on spreadsheets for statistical purposes.

**Comment by anjosh** [comment]
Mr. Ross,

Do we know what kind of information was sent or just the numbers?
Do you suggest include the statement from Josh?

*Olga Majitova, Esq.*
HSA-UWC
212-719-4980
212-719-4996
4W 43rd St
New York NY 10036

**Comment by noahross72** [comment]
I think they send in all the data…

**Comment by anjosh** [comment]
Olga,

You're right that statement is not true. We will use the information for
statistical purposes.

**Comment by anjosh** [comment]
After discussion with Olga and Noah. We determined that the privacy policy is good. I'll close this once we have the Policy on the website.
